docs/resources/release_policy.md

```diff
@@ -0,0 +1,89 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "launchdarkly_release_policy Resource - launchdarkly"
+subcategory: ""
+description: |-
+  Provides a LaunchDarkly release policy resource.
+  This resource allows you to create and manage release policies within your LaunchDarkly organization.
```

Comment:
If it's possible to edit this, let's add that release policies are in beta. This could also be a good place to link to our public docs.